src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -96,7 +96,12 @@ public PrintStreamWithDate(String logfile) throws FileNotFoundException, Unsuppo
 
         @Override
         public void println(String string) {
-            super.println("[" + LocalDateTime.now().format(LOG_DATE) + "] " + string);
+            StackTraceElement[] stack = Thread.currentThread().getStackTrace();
+            string = "[" + LocalDateTime.now().format(LOG_DATE)
+                    + (stack.length > 2 ? " | " + stack[2] : "")
```

Comment:
I think using the whole stacktrace is too long, most logs end up with things like:
`com.github.manolo8.darkbot.Bot.main(Bot.java:51)`
I think we should shorten the package if it's from the bot (ie: `com.github.manolo8.darkbot` -> `db` or `eu.darkbot.api` ->`dbapi`)